src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService check = new UserService();
+        if (check.findByEmail(email) != null) {
+            User usr = check.findByEmail(email);
```

Comment:
```suggestion
            User user = userService.findByEmail(email);
```
let's move this variable before `if`